replacements/README.md

```diff
@@ -0,0 +1,116 @@
+#  OpenRCT2/OpenMusic/replacements
+
+This page is the main hub for the soundtrack of OpenRCT2.
+
+> What are the different names for?  
+
+This is so people can use the original RCT soundtrack together with the OpenRCT2 soundtrack without interfering with one another. Is RCT2 not loaded in? Loaded parks will then fall back onto its OpenRCT2 equivalent.
+
+ There will be three asset packs:
+ 
+ * `rct2.music` - All the music from RCT2.
+ * `openrct2.music.cover` - Cover of non-copyright music, e.g. merry-go-round, pirate theme, ragtime, organ etc.
```

Comment:
What would be in the places of copyrighted music?

Reply:
openrct2.music.alternative would replace the copyrighted music aswell as be available for everyone to use as extra tracks if they have RCT2 installed.

Reply:
Currently it replaces them with our OpenRCT2 equivalent. For example, space style will play galaxy style. Techno style will play Acid Style. Dodgems will play `openrct2.music.dodgems`